QuizWhois.Domain/Entity/Question.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace QuizWhois.Domain.Entity
+{
+    public class Question
+    {
+        public long Id { get; set; }
+        public string QuestionText { get; set; }
+        public string CorrectAnswer { get; set; }
+
+        public Question(long id, string questionText, string correctAnswer)
+        {
+            Id = id;
```

Comment:
We should not be creating Ids by hand, auto-incrementing is a task for EF Core. It would be created on SaveChanges automatically